.travis.yml

```diff
@@ -36,8 +36,9 @@ install: travis_retry composer install --no-interaction --prefer-source
 before_script:
   - travis_retry composer self-update
   - travis_retry composer install --no-interaction --prefer-source
+  - if [[ $polyfill = 'true' ]]; then travis_disable_php_ext mcrypt; fi
```

Comment:
What is `travis_disable_php_ext mcrypt` ? The functionality from https://github.com/travis-ci/travis-ci/issues/4701 hasn't yet been implemented